menuflow/nodes/http_request.py

```diff
@@ -83,18 +83,27 @@ async def request(self, session: ClientSession) -> Tuple(int, str):
             f"node: {self.id} method: {self.method} url: {self._url} status: {response.status}"
         )
 
-        # Tulir and its magic since time immemorial
         try:
-            response_data = RecursiveDict(CommentedMap(**await response.json()))
+            serialize_response = await response.json()
         except ContentTypeError:
-            response_data = {}
-
-        if self._variables:
-            for variable in self._variables:
-                try:
-                    variables[variable] = response_data[self.variables[variable]]
-                except KeyError:
-                    pass
+            serialize_response = {}
+
+        if type(serialize_response) == dict:
```

Comment:
The better is use `isinstance` for validate variable type:

e.g

- `if isinstance(serialize_response, dict):`
- `if isinstance(serialize_response, str):`